docs/cudf/source/user_guide/io.md

```diff
@@ -1,10 +1,17 @@
-I/O Supported dtypes
-====================
+# Input / Output
 
-The following table lists are compatible cudf types for each supported IO format.
+This page contains Input / Output related APIs in cuDF.
 
-.. rst-class:: io-supported-types-table special-table
+## I/O Supported dtypes
+
+The following table lists are compatible cudf types for each supported
+IO format.
+
+<div class="special-table-wrapper" style="overflow:auto">
+
+```{eval-rst}
 .. table::
+    :class: io-supported-types-table special-table
     :widths: 15 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10
 
     +-----------------------+--------+--------+--------+--------+---------+--------+--------+--------+--------+-------------------+--------+--------+---------+---------+
```

Comment:
I like how GitHub thinks you only moved this and 2 more files while you also made similar file movements in this PR 🤣

Reply:
Yeah what the heck is that about...